docs/manuals/core/entities/scenario-cycle-mgt.md

```diff
@@ -98,6 +98,23 @@ propagated to the nested pipelines, tasks, data nodes, and jobs if they are not
 
 If there is more than one scenario in the cycle, an error will be raised if you try to delete a primary scenario. You must promote another scenario as primary before deleting the primary one. On the other hand, if there is only one scenario in the cycle, the primary scenario can be deleted, which will also delete its cycle.
 
+# Compare scenarios
+
+You can compare two or more scenarios, created from the same scenario configuration, by using `taipy.compare_scenarios()` which takes two or more scenarios as the parameters. You can also provide the data node config id to specify which data nodes from the scenarios you want to compare through the parameter _**data_node_config_id**_.
```

Comment:
"which takes two or more scenarios as the parameters" 
->
". This function accepts two or more scenarios to be compared,"

Reply:
Remove the "through the parameter _**data_node_config_id**_" part